### PR TITLE
Token now sends only the public sale tokens to crowdsale contract

### PR DESCRIPTION
**Pull request by matt-lough**
- Clean up contracts
- Updated tests
- Added helper variables to show vesting amounts

### CODE REVIEW

File: test/pauseTokenTests.js [hunk 1]
```diff
@@ -15,17 +15,16 @@ require('chai')
 
 const expect = require('chai').expect
 
-contract('polyTokenPause', async function([miner, owner, investor, wallet]) {
+contract('polyTokenPause', async function([miner, owner, investor, investor2, wallet, presale_wallet]) {
```
Comment (OperationNine) [inline test/pauseTokenTests.js]: investor2? maybe we could call this something else to make it more tidy. any suggestions?

Comment (OperationNine) [reply test/pauseTokenTests.js]: contributor?

Comment (matt-lough) [reply test/pauseTokenTests.js]: 👀

File: contracts/PolyMathToken.sol [hunk 2]
```diff
@@ -14,16 +14,24 @@ contract PolyMathToken is PausableToken, BurnableToken {
...
+    balances[msg.sender] = INITIAL_SUPPLY.sub(PRESALE_SUPPLY);
   }
 
   function setOwner(address _owner) onlyOwner {
     pause();
-    balances[owner] = 0;
+    balances[owner] = INITIAL_SUPPLY.sub(PUBLICSALE_SUPPLY);
     owner = _owner;
-    balances[owner] = INITIAL_SUPPLY;
+    balances[owner] = PUBLICSALE_SUPPLY;
```
Comment (Janther) [inline contracts/PolyMathToken.sol]: logic is correct but difficult to read

File: test/tokenTests.js [hunk 3]
```diff
@@ -89,9 +89,9 @@ contract('polyToken', async function(accounts) {
 
   // ############## ERC20 TESTS ##############
 
-  let amount = new web3.BigNumber(1000000000000000000000000000);
+  let amount = new web3.BigNumber(850000000000000000000000000);
```
Comment (rstormsf) [inline test/tokenTests.js]: something like 10**18 * 85 or similar would be much easier to read

**Pull request merged**